3-Soru3.py

```diff
@@ -0,0 +1,18 @@
+
+
+class Name():
+    def __init__(self):
+        self.cust_name = ""
+        self.cust_name2 = ""
+    def get_cust_info(self):
+        self.cust_name = input("Enter Your Name:")
+
+    def uppercase(self):
+
+        return self.cust_name.upper()

```

Comment:
uppercase metodu `self.cust_name.upper()`'i return etmemeli, print etmeli.
```suggestion
        print(self.cust_name.upper())

```